### PR TITLE
Make 404.html a 404

### LINKED ISSUE #28: /404 is an HTTP/200
Thank you Netlify!

https://conedevelopment.com/404 HTTP/200 👉🏻 indexable by search engines
https://conedevelopment.com/404-knot-fund HTTP/404

What a joy. 🕹️

### PR DESCRIPTION
**Pull request by szepeviktor**
Resolves #28

☣️ This may brake the Internet!!!
It needs testing.

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /en/* /404.html 404
 /hu/* /hu/404.html 404
+/404 /404.html 404
```
Comment (szepeviktor) [inline _redirects:1]: Maybe this should the first line ...

**Comment by szepeviktor** [comment]
https://deploy-preview-29--cone-development-site.netlify.app/hu/404 is still a HTTP/200

It must be an ordering issue.
I give up. @adamlaki Please carry on!

**Comment by adamlaki** [comment]
Shouldn't a noindex meta be better at the 404 template?

Also, this can be a problem on other sites too so how I didn't stumble upon this?

**Comment by szepeviktor** [comment]
> Also, this can be a problem on other sites too so how I didn't stumble upon this?

I spend countless hours on unimportant stuff. So I see every byte with my laser eyes 👁️

**Comment by szepeviktor** [comment]
@adamlaki I let you fix Netlify 🪰

**Pull request closed without merging**